### PR TITLE
Adds construction graphs for shuttle structures

### PR DESCRIPTION
**Pull request by IamVelcroboy**
Changelog:

- Adds proper construct/deconstruct graphs for shuttle windows
- Adds construct/deconstruct graphs for shuttle walls
- Adds deconstruct graphs for diagonal shuttle walls (construct doesn't rotate as it should and may need to be done another way{by not using base girder graph} if we want them)
- Adds interior shuttle walls and associated construct/deconstruct graphs

swallscompare
swalls

:cl: Velcroboy
- add: added constructable/deconstructable shuttle walls/windows!

### CODE REVIEW

File: Resources/Prototypes/Entities/Structures/Walls/girders.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@
   components:
   - type: Transform
     anchored: true
-    noRot: true
+    noRot: false
```
Comment (Elijahrane) [inline Resources/Prototypes/Entities/Structures/Walls/girders.yml:9]: are you certain this will not cause issues for other walls?

Comment (IamVelcroboy) [reply Resources/Prototypes/Entities/Structures/Walls/girders.yml:9]: When I tested on my end there wasn't any funny issues I could find. Magil said he would test it on his end too. Just waiting to see what he says.

**Comment by IamVelcroboy** [comment]
OK, I just did a quick test to make sure no errors and construct/deconstruct worked. Everything should be good here.

**Comment by Elijahrane** [comment]
I'll need a bit longer to think about this because we were moving towards removing shuttle construction...

**Comment by IamVelcroboy** [comment]
While the conversation about these started when shuttle building was still a thing, I think it's more about consistency and having walls that can be properly constructed/deconstructed and have appropriate HP. Currently you can deconstruct the windows but not the walls which is just odd.

Shuttle building itself, I would say, is much less about the walls and more about the parts(console, thrusters, gyros.) Without those, these are more for style and allowing sanctioned shuttles some extra/balanced playibility. At best?worst? these just allow pre-made shuttles to be altered by players. 

Interior walls would be nice for the consistency throughout ships and ship maps and create more of a exterior hull(higher HP) vs interior wall(lower HP) dynamic. 

One thing though that does _need_ to be merged if the rest isn't, is the shuttle window changes. Because currently they can be deconstructed in like 2 seconds with a screwdriver. Also a fix was added for missing destruction sounds in rWalls.

I guess that's my take on it. I don't particularly think it encourages shittle building any more/less but rather just creates consistency throughout the game.

**Comment by Vordenburg** [comment]
I feel like there's two possible ways to go with this regarding consistency. Either give the walls/windows proper graphs, or make windows not deconstructible. I can see good points for going either way.

Making them have graphs means the cargo shuttle can be repaired with shuttle walls if it gets hit by a meteor.

Making all shuttle parts not constructible/deconstructible makes things cleaner from the standpoint of removing shuttle construction completely and will cut down on questions about "how do I build a thruster" or PRs aiming to make them buildable.

**Comment by Elijahrane** [comment]
shuttle maintenance / repair sounds good to me. if shuttle construction continues to be an issue it can be clamped down on via other methods

**Pull request merged**